src/main/java/mate/jdbc/service/CarService.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarService {
```

Comment:
Where are `addDriverToCar()` and `removeDriverFromCar()`?

Reply:
You previously recommended to delete this line, I thought that you mean to delete the entire method